### ISSUE TITLE
Lack of Documentation

### ISSUE THREAD

**Issue by malhobayyeb** [GitLiveApp/firebase-kotlin-sdk#686]
I'm trying to integrate the firebase-kotlin-sdk into my Kotlin Multiplatform project, but I'm finding it challenging due to the lack of documentation and tutorials.

**Comment by ksharma-xyz** [comment]
Agree, I have been able to integrate Firebase Crashlytics and Firebase Performance. But it is very hard to integrate Firebase Analytics as there is no documentation about it.

If it maybe of any help, feel free to look at this PR, that's how I did it - https://github.com/ksharma-xyz/Krail/pull/434/files

**Comment by ksharma-xyz** [comment]
I was able to make analytics work. Here's a PR 
- https://github.com/ksharma-xyz/Krail/pull/460
- https://github.com/ksharma-xyz/Krail/pull/457

It takes some time (couple hours) before analytics events data is visible on Firebase.

**Comment by ksharma-xyz** [comment]
Update for analytics:
Learned that Debug View is Firebase helps to check real time events, but it needs to enabled.  Here's a guide on how to enable it https://support.google.com/analytics/answer/7201382?hl=en&utm_id=ad

**Comment by AffiTheCreator** [comment]
Multiplatform is already difficult to grasp , lack of documentation for someone that is just starting with KMP, even with years of experience is a taunting exercise.
Still great work, and it's been very useful, when I know what I'm doing with it.

**Comment by bpappin** [comment]
I second this, its been wasting a lot of time trying things and trying to gets from other documentation.

**Comment by KatieBarnett** [comment]
+1 to this, I am not sure which dependencies I need and what I need for each platform. There are some old (2+ years) blog posts on the topic but I am not sure if they are still relevant.
